CHANGELOG.md

```diff
@@ -61,7 +61,7 @@
 - **activitypub:** add target_actor_id for create activity to broadcast post
   reply
   ([0128a21](https://code.castopod.org/adaures/castopod/commit/0128a21ec55dcc0a2fbf4081dadb4c4737735ba1))
-- **activitypub:** allow cors on get requests for routes exposing acitivitypub
+- **activitypub:** allow cors on get requests for routes exposing activitypub
```

Comment:
some projects don't like changing changelogs, happy to drop this or anything else.